src/pages/graphql/schema/orders/mutations/confirm-cancel-order.md

```diff
@@ -0,0 +1,71 @@
+---
+title: confirmCancelOrder mutation
+---
+
+# confirmCancelOrder mutation
+
+The `confirmCancelOrder` mutation allows a guest to cancel an order, passing its encoded identifier and a confirmation key.
+
+A guest customer can cancel an order only if the following conditions are true:
+
+ * The guest has previously placed the order.
+ * The `order_cancellation_enabled` configuration setting is `true`.
```

Comment:
```suggestion
 * The guest customer has previously placed the order.
 * The `order_cancellation_enabled` configuration setting is `true`.
```
I would also use `guest customer` in all other places where is referenced, just for consistency.